### PR TITLE
[11.0] partner_duplicate_mgmt

### PR DESCRIPTION
**Pull request by yasmineelmrini**

### CODE REVIEW

File: partner_duplicate_mgmt/init_hook.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
-# © 2016 Savoir-faire Linux
```
Comment (ventilooo) [inline partner_duplicate_mgmt/init_hook.py]: 2016-2017 plutôt ?

File: partner_duplicate_mgmt/init_hook.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
-# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+# © 2017-2018 Savoir-faire Linux
```
Comment (ventilooo) [inline partner_duplicate_mgmt/init_hook.py]: 2016-2018 plutôt ?

File: partner_duplicate_mgmt/wizard/base_partner_merge_automatic_wizard.py [hunk 3]
```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-# © 2016 Savoir-faire Linux
+# © 2017-2018 Savoir-faire Linux
```
Comment (ventilooo) [inline partner_duplicate_mgmt/wizard/base_partner_merge_automatic_wizard.py:2]: 2016-2017 plutôt ?

File: partner_duplicate_mgmt/init_hook.py (hunk 2 above)
Comment (ghost) [reply partner_duplicate_mgmt/init_hook.py]: @ventilooo no, the module was developped on september 2017

**Comment by ghost** [comment]
@yasmineelmrini when migrating a module to another Odoo version, please avoid as possible esthetical changes. Only change what needs to be changed in version 11.0.

File: partner_duplicate_mgmt/models/res_partner.py [hunk 4]
```diff
@@ -166,14 +173,14 @@ def create(self, vals):
...
         res = super(ResPartner, self).write(vals)
-        if 'name' in vals or 'firstname' in vals or 'lastname' in vals:
-            self._update_indexed_name()
 
         if (
             'parent_id' in vals or 'name' in vals or
             'lastname' in vals or 'firstname' in vals or
             'company_type' in vals or 'is_company' in vals
         ):
+            self._update_indexed_name()
```
Comment (ghost) [inline partner_duplicate_mgmt/models/res_partner.py:180]: looks to me like a bug fix that affects 10.0 as well ?

File: partner_duplicate_mgmt/wizard/base_partner_merge_automatic_wizard.py [hunk 5]
```diff
@@ -59,15 +59,6 @@ def _merge(self, partner_ids, dst_partner):
...
         partner_ids = Partner.browse(partner_ids).exists()
 
-        # check only admin can merge partners with different emails
-        if (
-            SUPERUSER_ID != self.env.uid and
-            len(set(partner.email for partner in partner_ids)) > 1
-        ):
-            raise UserError(_(
-                "All contacts must have the same email. Only the "
-                "Administrator can merge contacts with different emails."))
```
Comment (ghost) [inline partner_duplicate_mgmt/wizard/base_partner_merge_automatic_wizard.py]: looks like a bug in version 10.0 as well

File: partner_duplicate_mgmt/models/res_partner.py (hunk 4 above)
Comment (yasmineelmrini) [reply partner_duplicate_mgmt/models/res_partner.py:180]: No I don't think so, I can't remember now why i did this, but I'm sure that it was specific to 11.0. I'll think about it and justify my answer later.

**Pull request merged**